### PR TITLE
Use Solution string representation in Result string representation

### PR DESCRIPTION
**Pull request by leonlan**
This PR uses `Solution`s string representation to print the routes in the summary/string representation of `Result`.

### CODE REVIEW

**Comment by leonlan** [comment]
I just noticed this while reviewing the new notebooks. Will become handy when we change the Solution string representation, for example when we include vehicle type information.

File: pyvrp/Result.py [hunk 1]
```diff
@@ -95,9 +95,6 @@ def __str__(self) -> str:
             "Routes",
             "------",
         ]
-
-        for idx, route in enumerate(self.best.get_routes(), 1):
-            if route:
-                summary.append(f"Route {idx:>2}: {route}")
+        summary.extend(str(self.best).splitlines())
```
Comment (N-Wouda) [inline pyvrp/Result.py:98]: Or,
```suggestion
            "Routes",
            "------",
            str(self.best),
        ]
```
?

Comment (leonlan) [reply pyvrp/Result.py]: Yea this works!

**Review by N-Wouda — APPROVED** [review]
LGTM!

**Pull request merged**